compechem/calculators/tools.py

```diff
@@ -0,0 +1,73 @@
+import os
+from rdkit import Chem
+
+
+def generate_inchikey(molfile):
+    mol = Chem.MolFromMolFile(molfile)
+    inchikey = Chem.MolToInchiKey(mol)
+    return inchikey
```

Comment:
Anche questo lo mettere come metodo della classe `Molecule`, insieme a `generate_inchi`, ed entrambi metodi potrebbero venire eseguiti in automatico alla creazione dell'istanza. Magari in un blocco try/except, così non diventa bloccante nel caso in cui la generazione dell'InChi fallisse.

Reply:
Dovrei modificare il codice e fargli fare un single point con xTB alla creazione dell'istanza, visto che per generare Inchi/Inchikey abbiamo bisogno del file .mol

È molto rapido per una singola molecola, dobbiamo solo considerare se vale la pena farlo per tutte le molecole ad ogni step